src/commands/redis_cmd.cc

```diff
@@ -858,6 +870,9 @@ class CommandCAS : public Commander {
         return Status(Status::NotOK, errInvalidSyntax);
       }
     }
+    if (ex_exist && px_exist) {
```

Comment:
```suggestion
    if (ex_exist + px_exist > 1) {
```

Keep patten consistent.